### ISSUE TITLE
Gaussian blur functions.. keep or deprecate?

### ISSUE THREAD

**Issue by gregbubnis** [focolab/image-processing#6]
I wanted to put up for discussion whether the functions `gaussian2d` and `gaussian3d` are giving us any utility beyond what we would get from calling an existing library. Maybe they should be deprecated in favor or something such as `scipy.ndimage.filters.gaussian_filter()`

The source code is here:
https://github.com/scipy/scipy/blob/47bb6febaa10658c72962b9615d5d5aa2513fa3a/scipy/ndimage/filters.py#L265

### notes
- I find that the calling syntax of `gaussian2d` and `gaussian3d` is confusing, shuffling together filter standard deviations and kernel dimensions. The order of the parameters does not match the data dimensions.
- The scipy function takes an ndarray, and a list of standard deviations with the same axis order. 
- The scipy function also automatically chooses a kernel size based on the standard deviations but allows an optional override. This seems like standard practice and would spare us from requiring the kernel size parameters.
- The scipy function also does ND convolution as a series of 1D convolutions which is similar to, but more general than, the way that `gaussian3d` does a 2D then 1D convolution.

**Comment by gregbubnis** [comment]
Since ```gaussian3d``` just came up in discussion, wanted to ping you both @Borchardt  @RaymondDunn for thoughts on this

**Comment by RaymondDunn** [comment]
I agree, especially that the syntax is a little confusing to remember. I'm on team deprecate.

**Comment by RaymondDunn** [comment]
just found another issue #9 that describes a difference between what the docstring says gaussian3d does, and what it really does.
for me this puts another coin on the "eliminate gaussian blur functions" side of the scale.
